appveyor.yml

```diff
@@ -98,6 +100,25 @@ for:
       - sh: "./aws_cli/bin/python -m pip install awscli"
       - sh: "PATH=$(echo $PWD'/aws_cli/bin'):$PATH"
 
+      - sh: "sudo apt-get -y install python3.6"
+      - sh: "sudo apt-get -y install python2.7"
+      - sh: "sudo apt-get -y install python3.7"
+      - sh: "sudo apt-get update"
+      - sh: "sudo apt-get -y install python3.8"
+
+      - sh: "which python3.8"
+      - sh: "which python3.6"
+      - sh: "which python3.7"
+      - sh: "which python2.7"
+
+      - sh: "PATH=$PATH:/usr/bin/python3.8:/usr/bin/python3.7"
```

Comment:
do 3.6 and 2.7 also need to be added to path?

Reply:
2.7 and 3.6 are already there: https://www.appveyor.com/docs/linux-images-software/#python

I am considering moving off of the pre-installed everything in favor how installing python versions through apt-get. This will make things a little more consistent in appveyor, but didn't want to go that deep here.